app/routing/Versions.scala

```diff
@@ -24,7 +24,9 @@ object Version {
 
```

Comment:
This [PR](https://github.com/hmrc/individual-losses-api/pull/448/files) also adds a "from" function  - will we need that here? 
```suggestion
def from(request: RequestHeader, orElse: Version): Version =
    Versions.getFromRequest(request).getOrElse(orElse)
```

Reply:
Yes, I missed it. Have added it in, thanks